src/backend/web/routes/log.js

```diff
@@ -0,0 +1,42 @@
+require('../../lib/config');
+const express = require('express');
+const path = require('path');
+const fs = require('fs').promises;
+
+const log = express.Router();
+
+log.get('/', async (req, res) => {
+  if (process.env.LOG_FILE) {
+    try {
+      const logs = await fs.readFile(
+        path.join(__dirname, '../../../../', process.env.LOG_FILE),
+        'utf8'
+      );
+
+      let formattedLines;
```

Comment:
Instead of reading the log file in and splitting it in memory, switch this logic to use a file stream.  Open a file stream to the log file and pipe it into response.  You can run into some bad performance issues if you buffer large files this way.  Streaming web servers are the solution.